osf_tests/test_user.py

```diff
@@ -2153,8 +2145,9 @@ def test_can_gdpr_delete_shared_nodes_with_multiple_admins(self, user, project_w
 
         user.gdpr_delete()
 
-        # The deleted user is still associated with the node, though their name still appears as 'Deleted User'
-        assert user.nodes.all().count() == 1
+        # The deleted user is no longer associated with the node, though their name still appears in logs ect as
+        # 'Deleted User'
+        assert user.nodes.all().count() == 0
```

Comment:
Why did this count change?  I would think it was because there were no QuickFilesNodes, but test comment leads you believe there should still be a user on the node.

Reply:
So I wrote this GDPR code with Steve and I think this test and comment were initially wrong and counting the QFN instead of not counting nodes. So the node being counted really shouldn't be. [These lines](https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/models/user.py#L1628-L1630) cover that use case.